sormas-e2e-tests/src/main/java/org/sormas/e2etests/enums/AreaTypeValues.java

```diff
@@ -20,12 +20,13 @@
 
 import java.util.Random;
 import lombok.Getter;
+import lombok.SneakyThrows;
 
 @Getter
 public enum AreaTypeValues {
-  URBAN("URBAN"),
-  RURAL("RURAL"),
-  UNKNOWN("UNKNOWN");
+  URBAN("Urban"),
+  RURAL("Rural"),
+  UNKNOWN("Unknown");
```

Comment:
no, please keep also the lower case values because they are used in other methods. Create also the uppercase value here, so you can make a division like: uiValue and apiValue, since the uppercase values are used in the api calls or other checks